### ISSUE TITLE
matrixInput: row names affect matrix extensibility

### ISSUE THREAD

**Issue by SarahWagner** [INWTlab/shiny-matrix#1]
The following example, where the row names are set to `TRUE` works perfectly, i.e. the matrix extends as I add data. 
```r
 matrixInput(
                 inputId = "dataMatrix",
                 class = "numeric",
                 value = matrix(),
                 copy = TRUE,
                 paste = TRUE,
                 cols = list(
                   names = TRUE,
                   extend = TRUE,
                   delta = 1,
                   editableNames = TRUE
                 ),
                 rows = list(
                   names = TRUE,
                   editableNames = TRUE,
                   extend = TRUE,
                   delta = 1
                 ) 
               )
```

As I change the `name` option in `rows` to `FALSE`, the matrix does not extend anymore. 

```r
 matrixInput(
                 inputId = "dataMatrix",
                 class = "numeric",
                 value = matrix(),
                 copy = TRUE,
                 paste = TRUE,
                 cols = list(
                   names = TRUE,
                   extend = TRUE,
                   delta = 1,
                   editableNames = TRUE
                 ),
                 rows = list(
                   names = FALSE,
                   editableNames = TRUE,
                   extend = TRUE,
                   delta = 1
                 ) 
               )
```

**Comment by aneudecker** [comment]
Error message:
![image](https://user-images.githubusercontent.com/5587462/58231836-3c034a80-7d38-11e9-9ebb-370de172be10.png)

**Comment by aneudecker** [comment]
fixed as of 0.1.3.

@SarahWagner can you check if it works now?
